### PR TITLE
update doc links and instructions in cost management sources

### PR DESCRIPTION
**Pull request by boaz0**
related to https://github.com/project-koku/koku-ui/issues/1366

Updating the documents links in Cost Management steps (OCP, Azure, Aws).

//cc @dayleparker @nlcwong @rvsia 

# AWS
## step 3
image

## step 5
image

# Azure
## step 3

image

## step 4 (updating link url)
image

## step 5 (updating link url)
image

# OCP

## step 3
image

## step 4
image

## step 5
image

## step 6
image

### CODE REVIEW

File: packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js [hunk 1]
```diff
@@ -11,9 +11,14 @@ import {
     ClipboardCopyVariant
 } from '@patternfly/react-core';
 
+const DOC_PREFIX = 'https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3';
```
Comment (boaz0) [inline packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js]: @rvsia - this constant is being used in many files, do you have an idea where I should add it so I won't duplicate it?

Thanks :smile:

Comment (rvsia) [reply packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js]: there is `utilities/stringConstants.js` file

Comment (boaz0) [reply packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js]: Cool! Thanks.

**Comment by rvsia** [comment]
@boaz0 With using of `<Text variant.a` you need to update the tests :boom:

**Comment by nlcwong** [comment]
AWS Step 5, introductory text: should add in "the" to the sentence -- "To grant permission to the cost management..."

Azure Step 3, add a space between the text and the link

OCP Step 3, first check box, first bullet: change from 4.2 to 4.3

OCP Step 4, 2nd line of introductory text, remove that entire line and then append the "Learn more" link the first line of introductory text.  The user should be able to go to the learn more link to find out where the operator service account is located,  also it is referenced in the first numbered bullet.

OCP Step 5, move the "Learn more" link up to the end of the introductory text of the step.
For example: "The Usage Collector connects to your cluster ... for Cost Management. " [Learn more.]()

OCP Step 6, I think there is a cut/paste error!  The first step in this step, remove the first line and the Learn more because that doesn't belong here.  So the line should be: "1. Open the crontab entry ... "

**Comment by boaz0** [comment]
@nlcwong done. Please review this again. Thanks.

## AWS step 5
image

## Azure step 3
image

## OCP 
### step 3
image

### step 4
image

### step 5
image

### step 6
image

**Comment by dayleparker** [comment]
Hey @boaz0, the updates look great so far!
A few things I spotted:

AWS step 3
- "set up an Amazon S3..." (not 'on')
- Report name: koku   (no full stop needed after koku)
- Maybe instead of 'you just created in the S3 bucket field" we could change it to "you just created below" -- this is similar to how the Azure wizard words it, with a bit less text.

AWS step 5
-  Edit "AWS identity access management" to "AWS Identity and Access Management (IAM)" in first line and first step. Also, is the * needed in the first step? For referenceL: https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-set-up.html 
- Should the bullet points be numbered steps like in other wizard steps?
- Last line - change 'compute' to 'complete'

Azure step 4
- edit "... Subscription ID and enter it below:" (remove 'further' and add a colon :  )

Azure step 5
- For consistency, should we edit "Create a daily export" to "Create daily export" in the left nav and title?
- Remove the full stops from end of these phrases: "Daily export for billing-period-to-date costs" and "Storage account name created earlier"

OCP step 3
- Should the "Learn more" be moved to the end of the first sentence for consistency? (optional)

OCP step 4
- "Cost Management" should be lower case --> "cost management"

OCP step 5
- (Much shorter format looks great!)
- "Cost Management" should be lower case --> "cost management"
- Edit to last line in screen to make it shorter, and needs a colon: "Enter the cluster identifier below:"

OCP screen 6
- "Cost Management" should be lower case --> "cost management"

Thanks so much Boaz, please let me know if you have any questions :)

**Comment by boaz0** [comment]
@dayle 

## AWS step 3
image


## AWS step 5
image

## Azure step 4

image


## Azure step 5

image


## OCP step 3

image


## OCP step 4

image


## OCP step 5

image


## OCP screen 6

image

**Comment by boaz0** [comment]
@rvsia please review again and merge if it's OK.

Thanks.

**Pull request merged**